ArcGISRuntimeSDKQt_CppSamples/Maps/IdentifyLayers/IdentifyLayers.cpp

```diff
@@ -110,61 +111,50 @@ void IdentifyLayers::connectSignals()
     const double tolerance = 12.0;
     const bool returnPopups = false;
     const int maxResults = 10;
-    m_mapView->identifyLayers(mouseEvent.position().x(), mouseEvent.position().y(), tolerance, returnPopups, maxResults);
-  });
 
-  // handle the identify results
-  connect(m_mapView, &MapQuickView::identifyLayersCompleted, this, [this](const QUuid&, const QList<IdentifyLayerResult*>& results)
-  {
-    // reset the message text
-    m_message = QString();
-    int i = 0;
+    m_mapView->identifyLayersAsync(mouseEvent.position(), tolerance, returnPopups, maxResults).then(this,
+    [this](QList<IdentifyLayerResult*> results){
+      // reset the message text
+      m_message = QString();
 
-    for (IdentifyLayerResult* result : results)
-    {
-      ++i;
-      // lambda for calculating result count
-      auto geoElementsCountFromResult = [] (IdentifyLayerResult* result) -> int
```

Comment:
Originally I had changed this because I thought there was a bug in how this was being performed, but after looking at it again I don't believe there is. I can put it back to how it was and just QFuturize it if we want to do that.

Reply:
I would keep it the same. Things like this can end up inadvertently changing subtle behavior.